sql/_05_plcsql/_01_testspec/_04_expression/_04_function_call/_03_builtin_function/_06_information_function/cases/04_04_03_06_bfn_info_inet.sql

```diff
@@ -0,0 +1,21 @@
+--+ server-message on
+
+-- normal: basic usage of a builtin function call
+
+create or replace procedure t () as
+begin
+    dbms_output.put_line(INET_ATON(NULL)); -- param NULL parse error for first paramA
+    dbms_output.put_line(INET_NTOA(NULL)); -- param NULL parse error for first paramA
```

Comment:
INET_NTOA(NULL) causes a runtime error in SQL statement "select INET_NTOA(NULL);" too.
This needs to be resolved in a separate Jira issue.